src/Test.java

```diff
@@ -2,10 +2,11 @@ public class Test {
 	public static void main(String[] args) {
```

Comment:
remove this test class , its not required for master branch

Reply:
i removed unused test class